test/FacilitatorConfig/constructor.test.ts

```diff
@@ -0,0 +1,46 @@
+import {assert} from 'chai';
+import Utils from '../../src/Utils';
+import * as path from 'path';
+import {FacilitatorConfig} from '../../src/Config';
```

Comment:
Please run linter for all the test files.

```suggestion
import {FacilitatorConfig} from '../../src/Config';
```
```suggestion
import { FacilitatorConfig } from '../../src/Config';
```